index.html

```diff
@@ -226,50 +228,63 @@ <h5 class="card-title">Learn React</h5>
   </div>
 
 
-  <!-- Scroll-to-top button -->
-  <button id="scrollToTopBtn" onclick="scrollToTop()" title="Go to top">Top</button>
   
-  <footer class="footer" style="background-color: #373abe; color: #ffffff;">
-    <div class="container py-5">
-        <div class="row">
-            <div class="col-md-4">
-                <h5><b>~About Us~</b></h5>
-                <p>Introducing the exceptional team at PulseZest, where creativity meets security to deliver outstanding Android software, web pages, web portals and many Business Solutions.</p>
-            </div>
-            <div class="col-md-4">
-                <h5><b>~Quick Links~</b></h5>
-                <ul class="list-unstyled">
-                    <li><a href="#" style="color: #ffffff;">Home</a></li>
-                    <li><a href="#" style="color: #ffffff;">About</a></li>
-                    <li><a href="#" style="color: #ffffff;">Services</a></li>
-                    <li><a href="#" style="color: #ffffff;">Contact</a></li>
-                </ul>
-            </div>
-            <div class="col-md-4">
-                <h5><b>~Contact Us~</b></h5>
-                <ul class="list-unstyled">
-                    <li>Email: contact@pulsezest.com</li>
-                    <li>Phone: +91-5813554123</li>
-                    <li>Address: Pashupati Vihar Colony, Bareilly, U.P</li>
-                </ul>
-            </div>
-        </div>
-    </div>
-    <div class="text-center py-3" style="background-color: #373abe;">
-        &copy; 2024 PulseZest. All rights reserved.
-    </div>
-   
-    <div class="container">
-      <div class="row">
-        <div class="col" style="height: 20px;"></div>
-      </div>
-    </div>
+  <footer>
+
+    <!-- Scroll-to-top button -->
+    <div class="foot_panel1" onclick="scrollToTop()" title="Go to top">back to Top</div>
   
   
+    <div class="foot_panel2">
+        <ul class="foot">
+             <p>~About Us~</p>
+             <a>Introducing the exceptional team at PulseZest
+               where creativity meets security to deliver 
+               outstanding Android software</a>
+               <a>web pages</a>  
+               <a>web portals and many Business Solutions</a>
+             
+        </ul>
+        
+        <ul class="foot">
+        <p>~Quick Links~</p>
+            <a>Home</a>
+            <a> About</a>
+            <a>Services</a>
+            <a>Contact</a>
+        </ul>
+      
+
+        <ul class="foot">
+        <p>~Contact Us~</p>
+        <a>Email: contact@pulsezest.com</a>
+        <a> Phone: +91-5813554123</a>
+        <a>Address: Pashupati Vihar Colony, Bareilly,up</a>
+   </ul >
+ 
+   <ul class="foot">
+    <p>~Related Content~</p>
+    <a>Chrome for Developers</a>
+    <a>Chromium updates</a>
+    <a>Case studies</a>
     
+</ul>
+    </div>
+    <div class="foot_panel3">
+        <div class="logo"></div>
+    </div>
+    <div class="foot_panel4">

```

Comment:
remove the extra padding space below the text of  © 2024 PulseZest. All rights reserved

![image](https://github.com/PulseZest-Inovation/intern/assets/88832979/408d3bec-ca10-4ba9-8ee1-1a954a81c2fe)